lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -406,11 +406,6 @@ module InternalConstants
                  list: [:none, :gzip],
                  :default => nil
 
-    # Whether valid entries should be written even if some other entries fail
-    # due to INVALID_ARGUMENT or PERMISSION_DENIED errors when communicating to
-    # the Stackdriver Logging API. This is highly recommended.
-    config_param :partial_success, :bool, :default => true
```

Comment:
Should we not still accept it but ignore it, for backward compatibility (in case someone turned it on in their config)?

Reply:
+1

Reply:
Done.

Reply:
Done.

Reply:
Can we add a TODO: Breaking change. Remove this flag in Logging Agent 2.0.0 release.

Reply:
Also, might want to add `:deprecated` and `:skip_accessor` (see https://docs.fluentd.org/plugin-development/api-config-types).

Reply:
PTAL